### ISSUE TITLE
What is the equivalent of LogSoftmax function in Keras?

### ISSUE THREAD

**Issue by kuldeepyadav** [fchollet/keras#4087]
I am implementing a LSTM network using Keras for sequence labelling task. The original implementation is in Lua which uses "nn.LogSoftMax()" function. Is there any alternative to implement this function in Keras because the library seems to have only softmax activation at this point of time. Thank you.

**Comment by nouiz** [comment]
I don't know aboug keras, but if it don't have it, making a subclass of the
softmax layer to call theano log_softmax instead of softmax would be easy.
